frontend/app/src/composables/api/assets/index.ts

```diff
@@ -31,10 +31,10 @@ export function useAssetsApi(): UseAssetApiReturn {
 
   const performUpdate = async (version: number, conflicts?: ConflictResolution): Promise<PendingTask> => {
     const response = await api.post<PendingTask>('/assets/updates', {
-      asyncQuery: true,
+      async_query: true,
       conflicts,
-      upToVersion: version,
-    });
+      up_to_version: version,
+    }, { skipSnakeCase: true });
```

Comment:
There should be also a test for the endpoint, can you update it so that we have a guard against accidentally breaking it in the future?

As for this case, maybe we can think of, and integrate a mechanism to skip snake_case only partially in the future. e.g. skipSnakeCase: ['conflicts'] which snake_cases everything except conflicts.